cmd/spire-server/cli/jwt/mint_test.go

```diff
@@ -74,11 +75,9 @@ func TestMintRun(t *testing.T) {
 
 	server := new(fakeSVIDServer)
 
-	spiretest.StartGRPCSocketServer(t, util.DefaultSocketPath, func(s *grpc.Server) {
-		svidpb.RegisterSVIDServer(s, server)
-	})
+	socketPath := filepath.Join(dir, "socket")
 
-	alternativeSocket := spiretest.StartGRPCSocketServerOnTempSocket(t, func(s *grpc.Server) {
+	spiretest.StartGRPCSocketServer(t, socketPath, func(s *grpc.Server) {
```

Comment:
As mentioned in the PR description, this test changed to always rely on a non-default socket location to prevent unit test failures if SPIRE server was running.